xml/deploy_cephadm.xml

```diff
@@ -24,6 +24,17 @@
   &ceph; services. You can perform this via the &ceph; command line interface
   (CLI) or partially via &dashboard; (GUI).
  </para>
+ <important>
+   <para>
+     It is important to note that the &ceph; community documentation uses the
+     <command>cephadm bootstrap</command> command during initial deployment.
+     When using &productname; &productnumber;, the &cephsalt; tool does this for
+     you and you should not run the <command>cephadm bootstrap</command>
+     command at any point during your deployment. Any &ceph; cluster deployment
+     manually using the <command>cephadm bootstrap</command> will be
+     unsupported.
+   </para>
+ </important>
```

Comment:
I"m being nitpicky, so use your discretion. Remove the passive voice and second person.

---
Note that the &ceph; community documentation uses the <command>**cephadm bootstrap**</command> during initial deployment.  The &cephsalt; tool calls the <command>**cephadm bootstrap**</command> and should not be run directly.  Any &ceph; cluster deployment manually using the <command>**cephadm bootstrap**</command> will be unsupported.